homeassistant/components/zwave_me/lock.py

```diff
@@ -0,0 +1,57 @@
+"""Representation of a doorlock."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.components.lock import LockEntity
```

Comment:
I need to revise my alphabet!
```suggestion
from homeassistant.components.lock import LockEntity
from homeassistant.config_entries import ConfigEntry
```